sass/_query-utils.scss

```diff
@@ -0,0 +1,60 @@
+// Query Utils
+// ===========
+
+
+// Fix Max [function]
+// ------------------
+/// Subtract `1px` or `0.01em` from max breakpoint values,
+/// to avoid media-query overlap.
+/// @access private
+/// @param {length} $value -
+///   The max value to adjust
+/// @return {$length} -
+///   The adjusted value
+@function _fix-max(
+  $value
+) {
+  @if unit($value) == 'em' {
```

Comment:
what if someone uses rem? I've seen it and done that before myself :)

Reply:
I think browsers dislike `rem` media-queries. Should we give you a warning, or treat it like `em` and let users sort it out?

Reply:
I used to use them since I couldn't find any good research on why not. I've not done so for a few years, but I guess if I were to use this and use rems, I'd like to either have a little heads up (warning). A link to why would be best, but probably not necessary.

Reply:
ok, here's a full accounting of how they work in different browsers: https://zellwk.com/blog/media-query-units/

For our tool, I'll allow all three equally, and treat them the same – then provide a recommendation in the docs, linking to Zell's article.